src/Doxyfile.in

```diff
@@ -1130,7 +1130,7 @@ HTML_STYLESHEET        = @GUDHI_DOXYGEN_COMMON_DOC_PATH@/stylesheet.css
 # list). For an example see the documentation.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_EXTRA_STYLESHEET  =
+HTML_EXTRA_STYLESHEET  = @GUDHI_DOXYGEN_COMMON_DOC_PATH@/stylesheet.css
```

Comment:
I don't know why, but this change makes the header drop down menus no more work:
[before](https://gudhi.inria.fr/doc/latest/)

![Screenshot from 2022-06-08 09-52-25](https://user-images.githubusercontent.com/10407034/172562923-7c343955-0d9f-4165-b826-8092dc98ba77.png)

[after](https://output.circle-artifacts.com/output/job/2af5cf4a-b80e-407a-accd-6429394e73d1/artifacts/0/doxygen/index.html) [may require a circleci acount]

![Screenshot from 2022-06-08 09-52-54](https://user-images.githubusercontent.com/10407034/172562967-721cfd7d-dcc2-4063-9dca-9162252bdd36.png)

This is also the case for #637